### PR TITLE
Morphed draggable, no error on hypo card play

### LINKED ISSUE #2302: Morphed cards cannot always be played
Newly drawn morphed cards can only be played only if they have been morphed before the player's turn.

Steps to reproduce:
1. Watch any replay
2. Play / discard for every player for turn 1
3. Try to play the newly drawn blank card of the first player: it cannot be played
4. Morph the card and try again: it cannot be played
5. Morph someone else's blank card
6. When it's that player's turn, the card can be played
7. The morphed card of the first player _can_ be played when he draws another card

Expected result: Every morphed card (unless blank) should be playeable

### PR DESCRIPTION
**Pull request by DrAnax**
Closes #2302

### CODE REVIEW

**Comment by Zamiell** [comment]
thanks

File: client/src/game/ui/hypothetical.ts [hunk 1]
```diff
@@ -88,11 +88,9 @@ export function send(hypoAction: ClientAction): void {
...
       const card = getCardOrStackBase(hypoAction.target);
       const { suitIndex, rank } = card.getMorphedIdentity();
-      if (suitIndex === null) {
-        throw new Error(`Card ${hypoAction.target} has an unknown suit index.`);
-      }
-      if (rank === null) {
-        throw new Error(`Card ${hypoAction.target} has an unknown rank.`);
+      if (suitIndex === null || rank === null) {
+        // Play or discard action could have been initiated from the keyboard
+        return;
```
Comment (timotree3) [inline client/src/game/ui/hypothetical.ts:93]: This seems like the wrong direction. Can we detect this in the keyboard logic and prompt for the desired identity?

Comment (DrAnax) [reply client/src/game/ui/hypothetical.ts:93]: You mean, like, prevent this in keyboard.ts?

Comment (timotree3) [reply client/src/game/ui/hypothetical.ts:93]: Yes, in the same way we prevent in in HanabiCard, by prompting for the desired identity

**Pull request merged**